platforms/android/library/src/main/java/io/element/android/wysiwyg/viewmodel/EditorViewModel.kt

```diff
@@ -72,7 +70,12 @@ internal class EditorViewModel(
 
         composer?.log()
 
-        update?.menuState()?.let { menuStateCallback?.invoke(it) }
+        update?.menuState()?.let {
```

Comment:
I'd use either:

```suggestion
        (update?.menuState() as? MenuState.Update)?.actionStates?.let { states ->
            actionStatesCallback?.invoke(states)
        }
```

Or:

```suggestion
        val menuState = update?.menuState()
        if (menuState is MenuState.Update) {
            actionStatesCallback?.invoke(menuState.actionStates)
        }
```